### PR TITLE
REPO: Update references to uf-mil repositories after renaming

### PR DESCRIPTION
**Pull request by whispercoros**
Some repository cleanup was done. References to software-common were updated to point to mil_common, references to Sub8 were changed to point to SubjuGator (yes, we are apparently still doing this...), and references to Navigator were changed to point to NaviGator. The only instance of "Navigator" that was not changed was the mission singleton as a capital G would not work well in camel case.

The bagging aliases were moved out of the `scripts/bash_aliases.sh` file as suggested by @DSsoto in #192. They will be handled by the `bmux` and `bag` commands which will soon reside in mil_common. The formatting of this file will be used as a template to create a bagging aliases file for SubjuGator during the next test day.

### CODE REVIEW

**Comment by DSsoto** [comment]
@mattlangford It's not worth the effort to do that for NaviGator quite yet, eventually.

File: readme.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 # NaviGator Autonomous Surface Vehicle
 
-This is where all of the project specific code for [NaviGator](http://www.navigatoruf.org) hosted. The platform is designed to compete in the bi-anual [Maritime RobotX Challenge](https://www.robotx.org). For more information on project specifics, vehicle operation, or contacting the current team lead, please see the [wiki](https://github.com/uf-mil/Navigator/wiki).
+This is where all of the project specific code for [NaviGator](http://www.navigatoruf.org) hosted. The platform is designed to compete in the bi-anual [Maritime RobotX Challenge](https://www.robotx.org). For more information on project specifics, vehicle operation, or contacting the current team lead, please see the [wiki](https://github.com/uf-mil/NaviGator/wiki).
```
Comment (DSsoto) [inline readme.md]: _bi-annual, bīˈany(o͞o)əl/ (adjective): occuring twice a year_

File: scripts/bagging_variables.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script will be sourced by the bmux command and the variables will be
+# used by the bag command. It is imperative that each variable is in the bag_*
+# namespace and contains a space diliniated list of ROS topics. The only two
+# exceptions to this rule are the BAG_ALWAYS and BAG_DIR variables.
+
+
+# Define topics that should be in every bag
+export BAG_ALWAYS="/odom /absodom /clock"
```
Comment (DSsoto) [inline scripts/bagging_variables.sh]: BAG_STD: shorter?

File: utils/navigator_msgs/msg/Alarm.msg [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Sub8 Alarm message. All alarms should be published on the /alarm topic
+# SubjuGator Alarm message. All alarms should be published on the /alarm topic
```
Comment (DSsoto) [inline utils/navigator_msgs/msg/Alarm.msg]: Can you leave a comment here saying this is deprecated and alarms should be ported to ROS alarms before further development is done?

**Comment by DSsoto** [comment]
It would also be nice to change the Semaphore script now to reflect these changes.

File: readme.md (hunk 1 above)
Comment (whispercoros) [inline readme.md]: Oh no, I pulled a Matt!

File: scripts/bagging_variables.sh (hunk 2 above)
Comment (whispercoros) [inline scripts/bagging_variables.sh]: Nobody should ever be typing this manually and BAG_ALWAYS just seemed more clear to me.

File: utils/navigator_msgs/msg/Alarm.msg (hunk 3 above)
Comment (whispercoros) [inline utils/navigator_msgs/msg/Alarm.msg]: Will do.

**Comment by whispercoros** [comment]
Semaphore is dead to me, I will be focusing entirely on Jenkins after ***The Great Merge*** is complete.

File: scripts/bagging_variables.sh (hunk 2 above)
Comment (DSsoto) [inline scripts/bagging_variables.sh]: That's fine.

**Pull request merged**